### PR TITLE
Merge refactoring into master

### PR DESCRIPTION
**Pull request by kjanei**
#1 test do not merge

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -58,9 +58,8 @@ io.on("connection", (socket) => {
       )
         gameState.dice[i].available = false;
     }
-    io.emit("resetCheckboxes");
 
-    addPoints();
+    gameState.accumulatedPoints += gameState.potentialRollScore;
```
Comment (kjanei) [inline index.js:62]: Why did you change this?

```code blcok```

File: index.js [hunk 2]
```diff
@@ -119,8 +118,7 @@ io.on("connection", (socket) => {
 
   // Adds up accumulated points and rerolls all dice for the current player
   socket.on("freeRoll", () => {
-    addPoints();
-    io.emit("resetCheckboxes");
```
Comment (kjanei) [inline index.js:123]: Question 2

File: public/js/test.js [hunk 3]
```diff
@@ -43,16 +43,6 @@ $(() => {
...
-  socket.on("resetCheckboxes", () => {
-    for (let i = 0; i < NUMBER_OF_DICE; i++) {
-      document.getElementById(`dice${i}`).checked = false;
-    }
-    document.getElementById("rollDice").disabled = true;
-    document.getElementById("cashDice").disabled = true;
-    document.getElementById("zilch").style.display = "none";
-    document.getElementById("freeRoll").style.display = "none";
-  });
-
```
Comment (kjanei) [inline public/js/test.js:55]: Range comment

**Review by kjanei — COMMENTED** [review]
Overall comment

**Comment by s-ung** [comment]
Looks good.
